pandas/core/resample.py

```diff
@@ -1593,6 +1614,34 @@ def _get_period_bins(self, ax):
         return binner, bins, labels
 
 
+def _validate_resample_deprecated_args(offset=None, base=None, loffset=None, **kwds):
+    if base is not None:
+        warnings.warn(
+            "'base' in .resample() and in Grouper() is deprecated.\n"
+            "The new arguments that you should use are 'offset' or 'origin'.\n\n"
+            '>>> df.resample(freq="3s", base=2)\n'
+            "\nbecomes:\n\n"
+            '>>> df.resample(freq="3s", offset="2s")\n',
+            FutureWarning,
+            stacklevel=3,
+        )
+        if offset is not None:
+            raise ValueError("offset and base cannot be present at the same time")
+
+    if loffset is not None:
+        warnings.warn(
+            "'loffset' in .resample() and in Grouper() is deprecated.\n"
+            "Here an example to have the same behavior than loffset:\n\n"
```

Comment:
Don't think `"Here an example to have the same behavior than loffset:\n\n"` adds much to this warning